core/src/main/java/org/axonframework/serialization/Serializer.java

```diff
@@ -72,14 +72,12 @@
      * SerializedObject with the given {@code type} is an instance of the returned Class.
      * <p/>
      * If a class cannot be resolved (i.e. because the class is not available on this JVM's classpath) this method
-     * throws UnknownSerializedTypeException.
+     * returns an {@link UnknownSerializedType} instance which proides access to the raw underlying data.
```

Comment:
Question: Shouldn't we change it to `should return` i.o. `returns`, as it is part of the implementation? The interface doesn't enforce this.

Reply:
Discussed, we keep this as is.